app/server/appsmith-server/src/main/java/com/appsmith/server/domains/Application.java

```diff
@@ -101,9 +101,11 @@ public class Application extends BaseDomain implements Artifact {
     private String slug;
 
     @JsonView({Views.Internal.class, Git.class})
+    @Column(columnDefinition = "jsonb")
```

Comment:
Unfortunately, we have to add _both_ the annotations togather for it to work. I couldn't figure out a way to have just one annotation do the job of both of these, but it's unlikely we ever want to use just one of these anywhere.

```suggestion
    @Type(CustomJsonType.class)
    @Column(columnDefinition = "jsonb")
```